hibernate-core/src/main/java/org/hibernate/boot/model/source/internal/hbm/ModelBinder.java

```diff
@@ -4221,7 +4227,7 @@ public void addAttributeBinding(Property attributeBinding) {
 
 		@Override
 		public void process() {
-			log.debugf( "Binding natural-id UniqueKey for entity : " + entityBinding.getEntityName() );
```

Comment:
The above pattern has two drawbacks:

- unnecessary string concatenation cost even the produced string is simply discarded later due to level disabling
- the produced string might contain magic chars (i.e. `%s`) and will explode during runtime